controllers/front/notify.php

```diff
@@ -99,6 +97,19 @@ class ClearpayNotifyModuleFrontController extends AbstractController
      * @var mixed $config
      */
     protected $config;
+    /**
+     * Default API Version per region
+     *
+     * @var array
+     */
+    public $defaultApiVersionPerRegion = array(
+        'AU' => 'v2',
+        'CA' => 'v2',
+        'ES' => 'v1',
```

Comment:
Only `'ES'` ?

Reply:
Yes, only one from europe is needed, 'ES' or other from Europe.